### PR TITLE
#11 dev

### PR DESCRIPTION
**Pull request by MKB-1**

### CODE REVIEW

File: routes/api/users.js [hunk 1]
```diff
@@ -1,9 +1,88 @@
...
+const User = require('../../models/User');
+const bcrypt = require('bcryptjs');
+const jwt = require('jsonwebtoken');
+const config = require ('config');
+const gravatar = require('gravatar');
 
 // @route  GET api/users
 // @desc   Test route
 // @access Public
 router.get('/', (req, res) => res.send('Users route'));
```
Comment (gaticosalas) [inline routes/api/users.js:15]: Add route comment

File: routes/api/users.js [hunk 2]
```diff
@@ -1,9 +1,88 @@
...
```
Comment (gaticosalas) [inline routes/api/users.js:14]: delete test route

**Review by gaticosalas — CHANGES_REQUESTED** [review]

**Review by gaticosalas — APPROVED** [review]
Great!

**Comment by loren589** [comment]
Should work fine!

**Pull request merged**